### PR TITLE
Bengroseclose/ch22/user submission redirects to root budget

### PR DESCRIPTION
**Pull request by BenjaminGroseclose**
Completed [ch22](https://app.clubhouse.io/budgetsquirrel/story/22/user-submission-redirects-to-root-budget-creation)

### CODE REVIEW

File: BudgetSquirrel.Api/ClientApp/src/app/home/home/home.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+  selector: "bs-home",
+  template: `
+    <!-- Unauthenticated -->
+    <bs-splash-page *isAuthenticated="false"></bs-splash-page>
+
+    <!-- Authenticated -->
+    <bs-budgets *isAuthenticated="true"></bs-budgets>
```
Comment (ianmann56) [inline BudgetSquirrel.Api/ClientApp/src/app/home/home/home.component.ts:10]: I don't see where "isAuthenticated" is coming from? How does this work?

File: BudgetSquirrel.Api/ClientApp/src/app/shared/services/account.service.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export class AccountService {
   }
 
   public logout(): Observable<boolean> {
+    console.log("Test");
```
Comment (ianmann56) [inline BudgetSquirrel.Api/ClientApp/src/app/shared/services/account.service.ts:42]: Hi Test.

File: BudgetSquirrel.Api/ClientApp/src/app/home/home/home.component.ts (hunk 1 above)
Comment (ianmann56) [reply BudgetSquirrel.Api/ClientApp/src/app/home/home/home.component.ts:10]: Ohhhh I see, this is that directive you created?

Comment (BenjaminGroseclose) [reply BudgetSquirrel.Api/ClientApp/src/app/home/home/home.component.ts:10]: Yeah it is a directive I created

File: BudgetSquirrel.Api/ClientApp/src/app/shared/services/account.service.ts (hunk 2 above)
Comment (BenjaminGroseclose) [reply BudgetSquirrel.Api/ClientApp/src/app/shared/services/account.service.ts:42]: I can remove that was playing around with the logout stuff.

**Comment by BenjaminGroseclose** [comment]
So the details page is from the accordion work I was planning. My thought was to separate them into two different component to make it easier to maintain. I can remove it for now but I think it would be a good idea to put the accordion details into its open section. 

Also I created the budget directory because I imagine it becoming a big part of our app and would be good like account or shared to have its own section.

**Comment by ianmann56** [comment]
> So the details page is from the accordion work I was planning. My thought was to separate them into two different component to make it easier to maintain. I can remove it for now but I think it would be a good idea to put the accordion details into its open section

Ok, yeah let's not worry about that in this PR. I would get rid of everything that's not just showing the page and redirecting to it. So you really should only be creating one extra component: the Bugets page that will contain everything on the main page.

> Also I created the budget directory because I imagine it becoming a big part of our app and would be good like account or shared to have its own section.

Ok sounds good.

**Comment by BenjaminGroseclose** [comment]
Just pushed an update to remove that component

**Pull request merged**